<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="13" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Подключение</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Список чатов:</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Отправить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="nameUserLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QTextBrowser" name="ChatTextBrowser"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="loginPushButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Все пользователи:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QListUsersWidget" name="usersListWidget">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QPushButton" name="CreateChatPushButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Создать чат</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>pushButton</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>nameUserLabel</zorder>
   <zorder>loginPushButton</zorder>
   <zorder>ChatTextBrowser</zorder>
   <zorder>usersListWidget</zorder>
   <zorder>CreateChatPushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>383</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QListUsersWidget</class>
   <extends>QListWidget</extends>
   <header location="global">CustomersWidgets/qlistuserswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
